packages/web-react/src/components/Button/useButtonAriaProps.tsx

```diff
@@ -26,7 +26,7 @@ export function useButtonAriaProps(props: AriaButtonProps<ElementType>): ButtonA
       role: 'button',
       href: elementType === 'a' && disabled ? undefined : href,
       target: elementType === 'a' ? target : undefined,
-      type,
+      type: elementType === 'a' && type === 'button' ? undefined : type,
```

Comment:
I think that this should be enough.

```suggestion
      type: elementType === 'button'  ? type : undefined,
```

Reply:
in this condition there will never be an elementType button. I'm just trying not to set the default property type "button" here, which is set here at the top. Anchor has type attribute but it is used to define the mime type of the target document.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-type